src/handlers.cpp

```diff
@@ -194,11 +194,21 @@ void ImpiTask::on_get_av_failure(CassandraStore::Operation* op,
     LOG_DEBUG("No cached av found for private ID %s, public ID %s - reject", _impi.c_str(), _impu.c_str());
     send_http_reply(HTTP_NOT_FOUND);
   }
+  else if (error == CassandraStore::CONNECTION_ERROR)
+  {
+    // If the cache error is a failure to connect to the local cassandra,
+    // then we want Sprout to retry the request to another Homestead (as this
+    // could be a local issue to the node). Send a 503
+    LOG_DEBUG("Cache query failed as unable to connect to local cassandra");
```

Comment:
Also really picky, but can this be "Cache query failed: unable to connect to local cassandra"? "[Failure]: [reason]" is what we use elsewhere (e.g. line 1322 below) and "[Failure] as [reason]" looks a bit odd to me.